src/store/validation/index.ts

```diff
@@ -0,0 +1,165 @@
+import Vue from 'vue'
+import {
+  ActionTree, GetterTree, Module, MutationTree,
+} from 'vuex'
+import { IRootState } from '@/store'
+import Ajv, { ValidateFunction, ErrorObject } from 'ajv'
+import { JSONSchema7 } from 'json-schema'
+import { IBlock, IFlow } from '@floip/flow-runner'
+import { isEmpty, get, forIn } from 'lodash'
+
+const ajv = new Ajv({ allErrors:true })
+const DEV_ERROR_KEYWORDS = [
+  'additionalProperties', // unwanted extra props
+  'required' // missing props
+]
+
+let validators = new Map<string, ValidateFunction>() //AJV validators, keys are types
+
+interface IIndexedString { [key: string]: string }
+
+export interface IValidationStatus {
+  isValid: boolean | PromiseLike<any>;
+  ajvErrors?: null | Array<ErrorObject>;
+}
+
+export interface IValidationState {
+  validationStatuses: { [key: string]: IValidationStatus }; //important context for future debug or testing, keys are index like `flow/flowId
+}
+
+export const stateFactory = (): IValidationState => ({
+  validationStatuses: {} as { [key:string]: IValidationStatus },
+})
+
+export const getters: GetterTree<IValidationState, IRootState> = {
+  /**
+   * Human readable errors, keys are index like `flow/flowId/.path/.to/.prop`.
```

Comment:
Hi hi, are the slashes truly present? I am under the impression that it should be more like: 
```
flow/{flowId}/.path.to.prop
```

The above should then match our deep-linking tasks. Can you confirm?